src/framework/defaults.js

```diff
@@ -21,6 +21,18 @@ const KEYS = {
 const COLORS = Symbol('colors');
 
 const DEFAULTS = {
+    unknown: {
```

Comment:
Why not use 'com.endlessm.dinosaurs.en' as default? Just a suggestion.

Reply:
I figured it would be better to have the default be equal to the default colors in the modular framework default theme. The choice of "encyclopedia" as the default logo is pretty arbitrary, though.